src/result.ts

```diff
@@ -0,0 +1,89 @@
+import { LooseFlowGraph } from "./export/bops/model";
+import { DEFAULT_FLAG_CATEGORY, flatFlags } from "./flags";
+import { Breadcrumbs, ComponentType, Flag, ResultData } from "./types";
+
+const SUPPORTED_DECISION_TYPES = [ComponentType.Checklist, ComponentType.Answer];
+
+export const getResultData = (
+  breadcrumbs: Breadcrumbs,
+  flow: LooseFlowGraph,
+  flagSet: Parameters<ResultData>[0] = DEFAULT_FLAG_CATEGORY,
+  overrides?: Parameters<ResultData>[1]
```

Comment:
nit: These two type declarations are pointing at the parameters of a function. Could they point at the types directly to make this more readable? Maybe more specific `ResultData` types need to be exported.